priv/repo/migrations/20160919005829_add_summaries_table.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Aprb.Repo.Migrations.AddSummariesTable do
+  use Ecto.Migration
+
+  def change do
+    create table(:summaries) do
+      add :verb, :string
+      add :summary_date, :date
+      add :total_count, :integer, default: 0
+      add :topic_id, :integer
+      timestamps
+    end
+  end
```

Comment:
Doesn't this need a uniqueness constraint? Otherwise, the race condition will become a real bug.